pegasus/sites.v3/code.org/public/js/admins_email_signup_form.js

```diff
@@ -0,0 +1,17 @@
+// Code provided from Google reCAPTCHA
```

Comment:
Could you link the actual site? I have a feeling we'll want to reference it often as none of us are familiar with this

Reply:
Yes definitely! Alek gave me this code though and said she got it from you, so I'm not sure what to link 😅

Reply:
Ha, let's see what I can find ...

Reply:
I really think Alek found this, but I had it in my history!
https://www.sfdcpoint.com/salesforce/salesforce-web-to-lead-form-with-recaptcha/

Looks like this is coming from a blog about Salesforce, which I hope is credible enough 🙈

Reply:
Haha I think that works! 😂 Adding now…

Reply:
Added!